demisto_sdk/commands/lint/linter.py

```diff
@@ -1306,3 +1346,285 @@ def _get_commands_list(self, script_obj: dict):
         except Exception:
             logger.debug("Failed getting the commands from the yml file")
         return commands_list
+
+    def _is_native_image_support_script(
+        self,
+        native_image: str,
+        supported_native_images: Set[str],
+        script_id: str,
+    ) -> bool:
+        """
+        Gets a native image name (flag) and checks if it supports the integration/script that lint runs on.
+
+        Args:
+            native_image (str): Name of the Native image to run on.
+            supported_native_images (set): A set including the names of the native images that support the
+                                           script/integration that lint runs on.
+            script_id (str): The ID of the integration/script that lint runs on.
+        Returns (bool): True - if the native image supports the integration/script that lint runs on.
+                        False - Otherwise.
+        """
+        if native_image not in supported_native_images:
+            # Integration/Script isn't supported by the requested native image
+            logger.info(
+                f"Skipping checks on docker for {native_image} - {script_id} is not supported by the "
+                f"requested native image: {native_image}."
+            )
```

Comment:
```suggestion
                f"Skipping checks on docker for {native_image} - {script_id} because it is not supported by the "
                f"requested native image: {native_image}."
            )
```

Reply:
As discussed off line - the actual log is clear enough as it is.